src/pages/Profile.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import StyledFieldset from '../components/StyledFieldset';
+
+export default function Profile() {
+  const profile = {
+    firstName: 'Joseph',
+    lastName: 'Tajaran',
+    email: 'joseph@tajaran.com',
+    pronouns: 'he/him',
+    skill: 'UX Design',
+  };
+
+  return (
+    <StyledFieldset>
+      <form>
+        <legend>
```

Comment:
There should only be one legend per fieldset, but since we're removing the form, please remove the legends as well.